spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -383,6 +391,11 @@ public SpoonRunner build() {
         converter = FileConverter.class, required = true) //
     public File testApk;
 
+    @Parameter(names = { "--e" },
```

Comment:
Isn't it a bit uncommon to use double-dashes for short (i.e. single-letter) options? Usually the single-letter version uses one dash, and the long version uses two dashes (see the "-h" and "--help" options).